R/data_codebook.R

```diff
@@ -455,6 +463,33 @@ print_html.data_codebook <- function(x,
 }
 
 
+#' @rdname data_codebook
+#' @export
+display.data_codebook <- function(object,
+                                  format = "markdown",
+                                  font_size = "100%",
+                                  line_padding = 3,
+                                  row_color = "#eeeeee",
+                                  ...) {
+  format <- .display_default_format(format)
+
+  fun_args <- list(
+    x = object,
+    font_size = font_size,
+    line_padding = line_padding,
+    row_color = row_color
+  )
```

Comment:
Maybe minor simplification, is it possible to add `...` directly in this list and therefore just have `do.call(<function>, fun_args)` below?